src/Component/FileImportButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+
+type FileImportButtonProps = {
+  id: string;
+  buttonText: string;

```

Comment:
I'd probably use `props.children` instead of a `buttonText` field, that way the caller looks like `<FileImportButton>Import LiveSplit file</FileImportButton>`

Reply:
Also means you can nest html in there if needed e.g. to put an Icon in front of the text label

Reply:
Updated